opentracing-tracerresolver/src/main/java/io/opentracing/contrib/tracerresolver/TracerResolver.java

```diff
@@ -83,15 +115,18 @@ public static Tracer resolveTracer() {
             LOGGER.finest("GlobalTracer is not found on the classpath.");
         }
 
+        if (classloader == null) {
```

Comment:
Could you move this inside the `!isDisabled` branch? Then there is no unnecessary call done if the resolution is actually disabled.

Reply:
Resolved. Thanks!